profiles/tests/test_views.py

```diff
@@ -62,3 +64,24 @@ def test_edit_ryan(self):
         p = Profile.objects.get(user=self.user)
         self.assertEqual(p.bitbucket_url, "zerg")
         self.assertEqual(p.gitlab_url, "zerg")
+
+    def test_view_with_favorite_packages(self):
+        category = Category.objects.create(
+            title="Test Favorite",
+            slug="test_favorite",
+            description="Category to test favorites",
+        )
+        package = Package.objects.create(
+            title="Test Favorite", slug="test_favorite", category=category
+        )
+        Favorite.objects.create(package=package, favorited_by=self.user)
+        self.assertTrue(
+            self.client.login(username=self.user.username, password=STOCK_PASSWORD)
+        )
+        url = reverse(
+            "profile_detail", kwargs={"github_account": self.profile.github_account}
+        )
+        response = self.client.get(url)
+        self.assertContains(response, "Profile for user")
+        self.assertContains(response, "Favorite packages")
```

Comment:
This assertion is failing because the profile probably does not have `share_favorites` set to `True`. We should set `self.profile.share_favorites=True` and save the profile before sending the request.

Reply:
You're right, that's the reason for the error, thanks!